frontend/src/logic/schedule-parser/schedule.parser.ts

```diff
@@ -58,26 +57,26 @@ export class ScheduleParser implements ScheduleProvider {
     if (!dataRow) {
       throw new Error("No metadata provided");
     }
-    // Assumption made, that days are always go after metadata
+    // + 1, because in first row goes metadata
     const daysRow = rowParsers[start + 1];
     const notSectionsRowsCountFromBeginning = 3;
     rowParsers = rowParsers.slice(start + notSectionsRowsCountFromBeginning);
     return [new MetaDataParser(dataRow, daysRow), rowParsers];
   }
 
-  private initSections(
+  private groupParsersBySections(
     schedule: DataRowParser[],
     metaData: MetaDataParser
-  ): [ChildrenInfoParser, ShiftsInfoParser, ShiftsInfoParser] {
+  ): Omit<Sections, "Metadata" | "ExtraWorkersInfo"> {
```

Comment:
Jeszce jedna fajna rzecz o której nie wiedziałem :D 
Ona pozwala powiedzieć, że ta funkcja musi zwracać interfejs który posiada wszystkie pola z interfejsu `Sections` oprócz `"Metadata","ExtraWorkersInfo"`

Reply:
Właśnie w szoku byłem jak to zobaczyłem. Sztos :D